src/common/camera_control.c

```diff
@@ -26,10 +26,7 @@
 
 #include <unistd.h>
 #include <stdlib.h>
-#if defined(__SUNOS__)
```

Comment:
#include <fcntl.h> is POSIX and LSB compliant see http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/fcntl.h.html. If there are developers using non posix compliant systems they should add #ifdef for those systems